cmd/client/capture.go

```diff
@@ -192,6 +194,7 @@ func addCaptureFlags(cmd *cobra.Command) {
 	cmd.Flags().IntVarP(&headerSize, "header-size", "", 0, fmt.Sprintf("Header size of packet used, default: %d", flow.MaxCaptureLength))
 	cmd.Flags().IntVarP(&rawPacketLimit, "rawpacket-limit", "", 0, "Set the limit of raw packet captured, 0 no packet, -1 infinite, default: 0")
 	cmd.Flags().BoolVarP(&extraTCPMetric, "extra-tcp-metric", "", false, "Add additional TCP metric to flows, default: false")
+	cmd.Flags().BoolVarP(&socketInfo, "socket-info", "", false, "Add additional Socket process informations to flows, default: false")
```

Comment:
nit: s/informations/information/  it doesn't take a 's' at the end

Reply:
done